### PR TITLE
PP-4617 - Disable auto deploy

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT
- Added a flag to deploy docs manually as we are not aiming to automate the process now

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
```
Comment (rhowe-gds) [inline deploy.sh]: Why remove `u`? looks like we don't expect any unbound variables in this script

Comment (kbottla) [reply deploy.sh]: It is an issue with virtualenv which is failing this script on `./venv/bin/activate: line 57: PS1: unbound variable`

Comment (rhowe-gds) [reply deploy.sh]: ugh, that's nasty.
in the case, I'd surround that line with
`set +u`
and
`set -u`

Comment (kbottla) [reply deploy.sh]: 👍 Updated to have it only around `./venv/bin/activate`

**Review by rhowe-gds — APPROVED** [review]

**Pull request merged**